src/pkg/inspectors/imagescanner/pkg/controller.go

```diff
@@ -143,48 +139,28 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 	}
 
 	// Assessment report.
-	cc := policy.Spec.Inspection.DeepCopy()
-	//cc.DataProvider.Credential = nil
-	report := &v1alpha1.AssessmentReport{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%s-%s", reportNamePrefix, time.Now().UTC().Format("20060102-1504-05")),
-			Namespace: *policy.Spec.WorkNamespace,
-			Annotations: map[string]string{
-				v1alpha1.CreationTimestampAnnotation: fmt.Sprintf("%d", time.Now().UTC().Unix()),
-				v1alpha1.OwnerReferenceAnnotation:    policy.Name,
-			},
-		},
-		Spec: v1alpha1.AssessmentReportSpec{
-			InspectionConfiguration: *cc,
-			NamespaceAssessments:    make([]*v1alpha1.NamespaceAssessment, 0),
-		},
-	}
-
-	// Set owner reference when necessary.
-	if policy.Spec.Inspection.Assessment.ManagedBy {
-		if err := ctrl.SetControllerReference(policy, report, c.scheme); err != nil {
-			return errors.Wrap(err, "set controller reference")
-		}
+	report := itypes.AssessmentReport{
+		NamespaceAssessments: make([]*itypes.NamespaceAssessment, 0),
 	}
 
 	// Use closure to warp the related parameters.
-	inspectFac := func() func(workload v1alpha1.Workload) *v1alpha1.WorkloadAssessment {
-		return func(workload v1alpha1.Workload) *v1alpha1.WorkloadAssessment {
+	inspectFac := func() func(workload wl.Workload) *itypes.WorkloadAssessment {
```

Comment:
itypes is short for inspector types.
Because the workloadAssessment is for image scanner only. So move down from v1alpha1.